markdown/events/2022/bytesize-resources-to-learn-nextflow.md

```diff
@@ -23,3 +23,59 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[15:49](https://www.youtube.com/watch?v=oO7rAp-QZOkt=949)
+One is to run the nf-core pipelines straight out of the box. Especially if you're a facility and there's an nf-core pipeline that fits your needs. One is to use the nf-core pipeline straight out of the box. Second is to run the nf-core pipelines with some modifications. Either you turn off some of the features or you add some features or modules on your own with some modifications, or create an entirely new pipeline from scratch using the nf-core template. Now using the nf-core template gives the advantage of making sure that they all have the same structure as the other nf-core pipelines. You can then adapt a lot of pieces, especially modules and subworkflows from other nf-core pipelines into your templated pipeline that you're creating. In that case, all you have to do will be to basically shop for modules in nf-core if you have all the modules available there or consider making one if it isn't there, so that you are giving back to the community as well. And in some cases, you'll have to keep some local modules.
+
+[17:10](https://www.youtube.com/watch?v=oO7rAp-QZOkt=1030)
+Once you plan all this, you can just build your pipeline from the ground up.
+
+"Through open discussion and collaboration among the community, it's possible to leverage the knowledge of experts across the world for the development of domain-specific pipelines and implementation of current best practice analysis methods."
+
+This is from the nf-core Nature paper, and I truly think these words hold value and I have seen it in practice. I have learned from this community a lot, so it is possible and we should keep doing this. And just so that you know, there's a new mentorships program being launched jointly by the Nextflow and nf-core community. They're hoping to organize mentor and mentee pairs, and especially reach underrepresented groups and areas to improve outreach. You'll find more details about this on the nf-core website. I think the current round is already done but they have a couple more rounds coming as well. With that, I just want to say thank you and welcome to the community. This is pretty much my setup that you can see here but I'm sure many of you will resonate with that as well. This is currently my short guide on resources to learn Nextflow. We'll soon have more improvements, but we'll be back with those again, but thank you so much everyone.
```

Comment:
```suggestion
This is from the nf-core Nature paper, and I truly think these words hold value and I have seen it in practice. I have learned from this community a lot, so it is possible and we should keep doing this. And just so that you know, there's a new mentorships program being launched jointly by the Nextflow and nf-core community. They're hoping to organize mentor and mentee pairs, and especially reach underrepresented groups and areas to improve outreach. You'll find more details about this on the nf-core website. I think the current round is already done but they have a couple more rounds coming as well. With that, I just want to say thank you and welcome to the community. This is pretty much my setup that you can see here but I'm sure many of you will resonate with that as well. This is currently my short guide on resources to learn Nextflow. We'll soon have more improvements, but we'll be back with those again, thank you so much everyone.
```